workers/generic-worker/main_test.go

```diff
@@ -41,15 +41,6 @@ func TestIdleWithoutCrash(t *testing.T) {
 	}
 }
 
-func TestRevisionNumberStored(t *testing.T) {
-	if !regexp.MustCompile("^[0-9a-f]{40}$").MatchString(revision) {
-		t.Fatalf("Git revision could not be determined - got '%v' but expected to match regular expression '^[0-9a-f](40)$'\n"+
-			"Did you specify `-ldflags \"-X github.com/taskcluster/generic-worker.revision=<GIT REVISION>\"` in your go test command?\n"+
-			"Try using build.sh / build.cmd in root directory of generic-worker source code repository.", revision)
-	}
-	t.Logf("Git revision successfully retrieved: %v", revision)
-}
-
```

Comment:
Yes, this test tests that the test binary was built correctly, not the release binary - but that is important for knowing that the test environment accurately reflects a release binary. In other words, this test is not useful for validating that the release has the revision number (the addition to `build.sh` to do this was a great idea to solve that problem, many thanks) - but it is still useful for ensuring the test binary works like the release binary. So please keep it.